### PR TITLE
Suricata - update to latest upstream version 5.0.1.

### PR DESCRIPTION
**Pull request by bmeeks8**
### Suricata-5.0.1
This updates the Suricata binary package to v5.0.1 for use on AMD64 architectures. The Release Notes for v5.0.1 can be viewed [here](https://suricata-ids.org/2019/12/13/suricata-5-0-1-released/).

### CODE REVIEW

File: security/suricata/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Created by: Patrick Tracanelli <eksffa@freebsdbrasil.com.br>
-# $FreeBSD$
+# $FreeBSD: head/security/suricata/Makefile 520857 2019-12-25 13:30:20Z amdmi3 $
```
Comment (rbgarga) [inline security/suricata/Makefile]: Please don't add svn macro expansions here on git

File: security/suricata/Makefile [hunk 2]
```diff
@@ -76,7 +76,7 @@ LUA_CONFIGURE_ON=		--enable-lua \
 				--with-liblua-includes=${LUA_INCDIR} \
 				--with-liblua-libraries=${LUA_LIBDIR}
 
-LUAJIT_LIB_DEPENDS=		libluajit-5.1.so:lang/luajit-openresty
+LUAJIT_LIB_DEPENDS=		libluajit-5.1.so:lang/luajit
```
Comment (rbgarga) [inline security/suricata/Makefile]: Please keep this change to use luajit-openresty. All pfSense ports that need it are using this variant

File: security/suricata/files/suricata.in [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# $FreeBSD$
+# $FreeBSD: head/security/suricata/files/suricata.in 454649 2017-11-21 18:55:28Z zeising $
```
Comment (rbgarga) [inline security/suricata/files/suricata.in]: Please don't add svn macro expansion here

File: security/suricata/Makefile (hunk 1 above)
Comment (bmeeks8) [reply security/suricata/Makefile]: Removed. This came over from upstream FreeBSD ports I guess.

File: security/suricata/Makefile (hunk 2 above)
Comment (bmeeks8) [reply security/suricata/Makefile]: Reverted my change. This also came over from FreeBSD-ports upstream.

File: security/suricata/files/suricata.in (hunk 3 above)
Comment (bmeeks8) [reply security/suricata/files/suricata.in]: Removed. Another instance that came over from FreeBSD-ports upstream.

File: security/suricata/files/suricata.in [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# $FreeBSD$
+# $FreeBSD:$
```
Comment (rbgarga) [inline security/suricata/files/suricata.in]: There is still an extra : here

File: security/suricata/Makefile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Created by: Patrick Tracanelli <eksffa@freebsdbrasil.com.br>
-# $FreeBSD$
+# $FreeBSD:$
```
Comment (rbgarga) [inline security/suricata/Makefile]: There is still an extra : here

File: security/suricata/files/suricata.in (hunk 4 above)
Comment (bmeeks8) [reply security/suricata/files/suricata.in]: Rmoved the extra colon.

File: security/suricata/Makefile (hunk 5 above)
Comment (bmeeks8) [reply security/suricata/Makefile]: Rmoved the extra colon.

**Comment by rbgarga** [comment]
As requested by @bmeeks8, do not merge it until he says it's OK

**Pull request closed without merging**